lib/src/AppRoute.dart

```diff
@@ -10,7 +10,7 @@ class AppRoute {
 
   final _route = <String, WidgetBuilder>{
     // view_post: (context) => ViewPost(title: "ViewPost"),
```

Comment:
- if which line didn't use, pls remove its 
- and I don't understand what to do in this line, pls to comment what is it to do for